build_web_compilers/mono_pkg.yaml

```diff
@@ -1,6 +1,6 @@
 dart:
 # TODO: Update to `dev` https://github.com/dart-lang/build/issues/2841
-- 2.10.0-110.0.dev
+- 2.10.0
```

Comment:
Consider also adding `dev` in here I think that issue is resolved?

Reply:
I'll follow up in a separate PR. I don't know if there has been a dev SDK published with the fix.